metal/loadbalancers/metallb/metallb.go

```diff
@@ -28,7 +28,14 @@ type LB struct {
 
 func NewLB(k8sclient kubernetes.Interface, config string) *LB {
 	var configmapnamespace, configmapname string
-	cmparts := strings.SplitN(config, ":", 2)
+	// it may have an extra slash at the beginning or end, so get rid of it
+	if strings.HasPrefix(config, "/") {
+		config = config[1:]
+	}
+	if strings.HasSuffix(config, "/") {
+		config = config[:len(config)-1]
+	}
+	cmparts := strings.SplitN(config, "/", 2)
 	if len(cmparts) >= 2 {
```

Comment:
nitpic: `>` doesn't really apply anymore

Reply:
Oh that is a good point! It doesn't break anything, and I don't want to hold this up when under real-world usage issues, so I am going to merge this in (as you already approved) and will get that in the next changes.